### PR TITLE
James styling

### PR DESCRIPTION
**Pull request by jameshmread**
Sorry this is huge
Major styling changes
Home page now shows whatever stores users have access to sales and differences along with total

### CODE REVIEW

File: app/Http/Controllers/HomeController.php [hunk 1]
```diff
@@ -15,6 +18,52 @@ public function __construct()
...
+    {
+        return Customer::all()->count();
+    }
+
+    public function totalSalesValue()
+    {
+        $storeSalesPrevious = [];
+        $storeSalesCurrent = [];
+        $multiStoresSalesDifference = [];
+        $rights = Auth::User()->GetRights();
```
Comment (ghost) [inline app/Http/Controllers/HomeController.php]: I think this should be Auth::user()->GetRights();

With a lowercase u

File: package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
     "jquery": "^3.1.1",
     "laravel-mix": "^1.0",
     "lodash": "^4.17.4",
-    "vue": "^2.1.10"
+    "vue": "^2.1.0"
```
Comment (ghost) [inline package.json]: why choosing the lower vue version?

Comment (jameshmread) [reply package.json]: sure! see update

File: app/Http/Controllers/HomeController.php (hunk 1 above)
Comment (jameshmread) [reply app/Http/Controllers/HomeController.php]: sure, see update

**Review by ghost — APPROVED** [review]

**Pull request merged**